sparse/unit_test/Test_Sparse_coo2crs.hpp

```diff
@@ -214,6 +214,11 @@ void doAllCoo2Crs(size_t m, size_t n) {
 }
 
 TEST_F(TestCategory, sparse_coo2crs) {
+  if constexpr (std::is_same_v<typename TestDevice::execution_space, Kokkos::Experimental::SYCL>) {
```

Comment:
Oh wait, does a check for SYCL as an execution space require that it is enabled? Should this block of code be enclosed in preprocessor guards e.g. `ifdef KOKKOS_ENABLE_SYCL`?

Reply:
Yeah, the CI failed for non-SYCL jobs, the new code blocks will need to be guarded with `#ifdef KOKKOS_ENABLE_SYCL`